### PR TITLE
Large network policy add parameter

### PR DESCRIPTION
**Pull request by qiliRedHat**

### CODE REVIEW

**Comment by qiliRedHat** [comment]
@paigerube14 PTAL

File: perfscale_regression_ci/scripts/network/large_network_policy.sh [hunk 1]
```diff
@@ -49,9 +53,9 @@ do
...
 done
 
-echo "Total flows in pods $total_policies"
+echo "======Total flows in pods $total_policies======"
 
-if [[ $total_policies -ge 5000 ]]; then
+if [[ $total_policies -ge $JOB_ITERATION ]]; then
   echo "======PASS======"
   echo "Deleting test objects"
   oc delete project -l kube-burner-job=large-network-policy
```
[the remark this answers is not in the document]
Comment (qiliRedHat) [reply perfscale_regression_ci/scripts/network/large_network_policy.sh:61]: @paigerube14 This is not covered in the common scripts. It's a very specific SDN network policy number verification from the flows. It's not so 'common' to be put in the common script I think.

[the remark this answers is not in the document]
Comment (paigerube14) [reply perfscale_regression_ci/scripts/network/large_network_policy.sh:61]: I meant use the delete function at the end of the loop there

[the remark this answers is not in the document]
Comment (qiliRedHat) [reply perfscale_regression_ci/scripts/network/large_network_policy.sh:61]: @paigerube14 Aha, I got you. Updated.

**Comment by qiliRedHat** [comment]
Also add a commit to change conc_job default parameters to the same as the origin script

**Pull request merged**